lib/phoenix_html/tag.ex

```diff
@@ -7,6 +7,7 @@ defmodule Phoenix.HTML.Tag do
   import Plug.CSRFProtection, only: [get_csrf_token: 0]
 
   @tag_prefixes [:aria, :data]
+  @csrf [param: "_csrf_token", method: "_method"]
```

Comment:
Please define two separate module attributes. If you do `@csrf[:method]`, it is still injecting the whole list at compile time.